pkg/kubelet/server/remotecommand/exec.go

```diff
@@ -51,8 +85,19 @@ func ServeExec(w http.ResponseWriter, req *http.Request, executor Executor, podN
 
 	err := executor.ExecInContainer(podName, uid, container, cmd, ctx.stdinStream, ctx.stdoutStream, ctx.stderrStream, ctx.tty, ctx.resizeChan)
 	if err != nil {
-		msg := fmt.Sprintf("error executing command in container: %v", err)
-		runtime.HandleError(errors.New(msg))
-		fmt.Fprint(ctx.errorStream, msg)
+		m := ExitError{}
```

Comment:
That works too :-)

On Thu, Aug 4, 2016 at 8:42 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In pkg/kubelet/server/remotecommand/exec.go
> <https://github.com/kubernetes/kubernetes/pull/26541#discussion_r73513635>
> :
>
> > @@ -51,8 +85,19 @@ func ServeExec(w http.ResponseWriter, req *http.Request, executor Executor, podN
> >
> >  	err := executor.ExecInContainer(podName, uid, container, cmd, ctx.stdinStream, ctx.stdoutStream, ctx.stderrStream, ctx.tty, ctx.resizeChan)
> >  	if err != nil {
> > -		msg := fmt.Sprintf("error executing command in container: %v", err)
> > -		runtime.HandleError(errors.New(msg))
> > -		fmt.Fprint(ctx.errorStream, msg)
> > +		m := ExitError{}
>
> I am replacing errorStream with func writeError(message string, rc *int).
> I have that for http, have to understand the websocket part now.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26541/files/0a10e2dcdaf86a90b5a718e96d44cc9e5286ca06#r73513635>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAABYmf5WdoWmeHew7puTchET1yR6Umwks5qcd4dgaJpZM4Ip8Gm>
> .
>